### ISSUE TITLE
error in AddOrSubtractTwoFloats

### ISSUE THREAD

**Issue by jedierikb** [EmKayDK/jstepper#7]
```
$('#txtTesting').jStepper({minValue:0, maxValue:23.999, minLength:2, minDecimals:3, maxDecimals: 3, decimalSeparator: '.'});
```

Enter value 0.999, then decrement one step.
If you look at the return value in AddOrSubtractTwoFloats for this operation, you will note it results in a NaN.  This is because you are trying to cast 

```Number( ".0-1" )``` 

It seems this function did not account for negative numbers properly.

**Comment by EmKayDK** [comment]
Thank you for the bug report. I will investigate and make a fix as soon as possible.

**Comment by EmKayDK** [comment]
This bug is now solved with the 1.5.0 release.

Thanks again :+1:
